systemtest/020-copy.bats

```diff
@@ -144,6 +144,14 @@ function setup() {
     diff -urN $shareddir $dir2/blobs
 }
 
+@test "copy: sif image" {
+    local localimg=dir:$TESTDIR/dir
+
+    run_skopeo copy sif:systemtest/testdata/busybox_latest.sif $localimg
```

Comment:
Danger: this will fail in gating tests, because those run while cd'ed to a different directory (which is and should be a perfectly fine thing to do). Suggestion:
```diff
diff --git a/systemtest/020-copy.bats b/systemtest/020-copy.bats
index 5d7ad6d5..d29a91ec 100644
--- a/systemtest/020-copy.bats
+++ b/systemtest/020-copy.bats
@@ -147,7 +147,7 @@ function setup() {
 @test "copy: sif image" {
     local localimg=dir:$TESTDIR/dir

-    run_skopeo copy sif:systemtest/testdata/busybox_latest.sif $localimg
+    run_skopeo copy sif:${TEST_SOURCE_DIR}/testdata/busybox_latest.sif $localimg
     run_skopeo inspect $localimg --format "{{.Architecture}}"
     expect_output "amd64"
 }
diff --git a/systemtest/helpers.bash b/systemtest/helpers.bash
index 2f57b215..a81c646e 100644
--- a/systemtest/helpers.bash
+++ b/systemtest/helpers.bash
@@ -1,6 +1,10 @@
 #!/bin/bash

-SKOPEO_BINARY=${SKOPEO_BINARY:-$(dirname ${BASH_SOURCE})/../skopeo}
+# Directory containing system test sources
+TEST_SOURCE_DIR=${TEST_SOURCE_DIR:-$(dirname ${BASH_SOURCE})}
+
+# Skopeo executable
+SKOPEO_BINARY=${SKOPEO_BINARY:-${TEST_SOURCE_DIR}/../bin/skopeo}

 # Default timeout for a skopeo command.
 SKOPEO_TIMEOUT=${SKOPEO_TIMEOUT:-300}
```
Download patch: [skopeo-1550.diff.txt](https://github.com/containers/skopeo/files/7943838/skopeo-1550.diff.txt)

IMPORTANT: carefully note and review the added `bin` in `SKOPEO_BINARY`. Looks like some time since the last I looked here, `make` writes into `bin`, not the root dir.

Reply:
Thank you!